apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TransactionTests.cs

```diff
@@ -144,7 +226,7 @@ public async Task AbortedThrownCorrectly()
             // connection 2 reads again -- abort should be thrown.
 
             // Note: deeply nested using statements to ensure that we dispose of everything even in the case of failure,
-            // but we manually dispose of both tx1 and connection1. 
+            // but we manually dispose of both tx1 and connection1.
```

Comment:
I'll create a separate PR to separate all the whitespace changes from actual changes.

Reply:
I've merged the change - if you rebase to HEAD now, the whitespace changes should go away.